tox.ini

```
@@ -219,3 +219,9 @@ commands =
     cp -r {distdir} {toxinidir}/dist # copy the wheel built by tox-wheel
     {envpython} -m pep517.build --source --out-dir={toxinidir}/dist {toxinidir}
     twine check {toxinidir}/dist/*.*
+
+
+[testenv:benchmarks-ci]
+deps =
+    pytest-codspeed
+commands = pytest --codspeed benchmarks/
```

For future cleaner diff... empty line at the end of the file

```suggestion
commands = pytest --codspeed benchmarks/

```